cep-recipe-jinja.md

```diff
@@ -0,0 +1,125 @@
+# Jinja functions in recipes
+
+The new recipe format has some Jinja functionalities. We want to specify what functions exist and their expected behavior.
+
+## The compiler function
```

Comment:
Since the discussion on stdlib further upthread died down, just leaving a note that this either needs to add the `stdlib` function, or propose an alternative way of how to realize the same functionality (and how it gets populated through the variant config).